config/config_initializer.go

```diff
@@ -54,6 +55,10 @@ func loadApplicationConfig(configFilePath string) (*Config, error) {
 	}
 	config.MetricCollector = metrics.NewCollector(config.Monitoring, logger)
 
+	// TODO disble this check when reliable acks are properly supported
```

Comment:
When is this going to be?

Reply:
https://github.com/teslamotors/fleet-telemetry/pull/167